Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Build deployable artifacts
+ARG BASE_CONTAINER=ubuntu:20.04
+FROM ${BASE_CONTAINER} as plotman-builder
+
+ARG CHIA_GIT_REFERENCE
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl jq python3 ansible tar bash ca-certificates git openssl unzip wget python3-pip sudo acl build-essential python3-dev python3.8-venv python3.8-distutils apt nfs-common python-is-python3
+
+RUN echo "cloning ${CHIA_GIT_REFERENCE}"
+RUN git clone --branch "${CHIA_GIT_REFERENCE}" https://github.com/Chia-Network/chia-blockchain.git \
+&& cd chia-blockchain \
+&& git submodule update --init mozilla-ca
+
+WORKDIR /chia-blockchain
+# Placeholder for patches
+RUN /bin/bash ./install.sh
+
+COPY . /plotman
+
+RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman && deactivate"] 
```

Comment:
Doesn't really matter, but activation won't outlast the shell so deactivation doesn't do much here.